### ISSUE TITLE
Landmark roles missing 

### ISSUE THREAD

**Issue by plannero** [umbraco/Umbraco-CMS#14481]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

10.4.0

### Bug summary

When trying to navigate the CMS using a screen reader, I cannot easily jump between the major parts of the interface (top menu, content tree, editing pane, ...). 

### Specifics

Landmark roles should be applied to all of the major parts of the page, according to best practice.
The easiest way to fix this would probably be to add appropriate landmark roles. 
Something like this:
`role="banner"` for the "hideBackofficeLogo" div
`role="navigation" aria-label="main menu"` for the "applications" div
`role="search"` for the "global-search" div
`role="navigation" aria-label="content tree"` for the #navigation div
`role="main" aria-label="edit panel"` for the ".umb-editor" div
...

Or, at the very least, the editor pane should begin with a heading for screen reader users, something like this `<h2 class="sr-only">Editor panel</h2>`

### Steps to reproduce

Log in to Umbraco. Use a screen reader. Try to navigate between landmarks. Not possible, because there are no landmarks.

### Expected result / actual result

Should be easy to list the landmarks present on the page.

**Comment by andr317c** [comment]
Hey! 
Thanks a lot for taking the time to post this issue. This is something we would like help with, I will mark this up as an up for grabs! 😄

**Comment by plannero** [comment]
Just found a duplicate (sort of) of this bug already reported about a year ago. See https://github.com/umbraco/Umbraco-CMS.Accessibility.Issues/issues/11 
This really shouldn't be such a big thing to fix. Just add a few attributes, that will cause no problems for anybody but be very useful to people using a screen reader.
